src/code/dialogs/opsDialog.js

```diff
@@ -28,12 +29,16 @@ const OpsDialog = WDialog.extend({
   addHooks: function () {
     WDialog.prototype.addHooks.call(this);
     window.map.on("wasabeeUIUpdate", this.update, this);
```

Comment:
I will sweep and rename wasabeeUIUpdate to wasabee:UI:update ... this was "update" in the first versions and, as you can imagine, it conflicted with several other known plugins.